projects/admin/src/app/utils/validators.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
You can perhaps use ng-core TimeValidator instead.

Reply:
see https://github.com/rero/ng-core/issues/320